stairway/src/test/java/bio/terra/stairway/FlightMapTest.java

```diff
@@ -153,4 +172,28 @@ public void serDesExceptions() {
     Assertions.assertThrows(
         JsonConversionException.class, () -> FlightMap.create(new ArrayList<>(), "garbage"));
   }
+
+  @Test
+  public void typeReferences() {
+    FlightMap flightMap = new FlightMap();
+
+    Map<String, List<UUID>> inputMap = new HashMap<>();
+
+    inputMap.put(
+        "mapKey",
+        new ArrayList<>(
+            Arrays.asList(
+                UUID.randomUUID(), UUID.randomUUID(), UUID.randomUUID(), UUID.randomUUID())));
```

Comment:
FYI the new collection APIs can streamline code like this
```suggestion
    Map<String, List<UUID>> inputMap =
      Map.of(
        "mapKey",
        List.of(UUID.randomUUID(), UUID.randomUUID(), UUID.randomUUID(), UUID.randomUUID()));
```